crates/zed/src/main.rs

```diff
@@ -1008,10 +1008,20 @@ async fn watch_languages(fs: Arc<dyn fs::Fs>, languages: Arc<LanguageRegistry>)
 
     let reload_debounce = Duration::from_millis(250);
 
+    #[cfg(not(target_os = "windows"))]
     let mut events = fs
         .watch("crates/zed/src/languages".as_ref(), reload_debounce)
         .await;
 
+    #[cfg(target_os = "windows")]
+    let mut events = {
+        let watch_path = Path::new("crates/zed/src/languages");
```

Comment:
`crates/zed/src/languages` is no longer exists, I think it can be removed.
https://github.com/zed-industries/zed/pull/8270

Reply:
Should I change it to `crates/languages`?

Reply:
Because `languages` will move into extentions, it may not.
Refs: https://github.com/zed-industries/zed/pull/8818#issuecomment-1979220241

Reply:
> Because `languages` will move into extentions, it may not. Refs: [#8818 (comment)](https://github.com/zed-industries/zed/pull/8818#issuecomment-1979220241)

Makes sense, thx!